add-on/src/lib/state.js

```diff
@@ -57,7 +57,8 @@ export function initState (options, overrides) {
   })
   Object.defineProperty(state, 'localGwAvailable', {
```

Comment:
I wonder if there is a specific reason we need to use defineProperty (e.g. we have logic enumerating on the state object and we don't want these to be enumerable, or prevent prop deletion, or prevent changing value?) or if this can be cleaned up in the future.

Reply:
Awesome, that's what I figured, thanks for the info